package.json

```diff
@@ -58,10 +56,5 @@
     ],
     "sourceMap": false,
     "instrument": false
-  },
-  "collective": {
-    "type": "opencollective",
-    "url": "https://opencollective.com/babel",
-    "logo": "https://opencollective.com/babel/logo.txt?reverse=true&variant=variant1&width=50"
```

Comment:
FYI, you can also simply remove the logo and it will show just the text in the post install like this ![](https://cl.ly/1k1O3j222Z2L/Screen%20Shot%202017-04-01%20at%208.19.09%20PM.png) 

Which is much more discrete than the version with the ASCII art:
![](https://cl.ly/0W2f0Q2U1m37/Screen%20Shot%202017-04-01%20at%208.19.58%20PM.png)